frontend/src/editor/automerge_websocket_provider.ts

```diff
@@ -1,4 +1,5 @@
 import { Observable } from 'lib0/observable';
```

Comment:
`lib0` is still an old dependency from the yjs-devs. Does it make since to keep it now that we decided against yjs or can we switch this to something from the stdlib or self-built?